src/test/java/com/hedera/exchange/ExchangeRateUtilsTestCases.java

```diff
@@ -68,4 +108,284 @@ public void getDecryptedEnvironmentVariableFromAWSTest() {
 		final String lambdaFunctioName = "exchange-rate-tool-lambda-integration";
```

Comment:
if this test is still not working/passing, either fix it or add a description in `@Disabled` about why it is disabled